src/editorial-source-components/CustomDropdown.tsx

```diff
@@ -0,0 +1,63 @@
+import { css } from "@emotion/react";
+import { space } from "@guardian/src-foundations";
+import { Option, Select } from "@guardian/src-select";
+import type { Option as OptionValue } from "../plugin/types/Element";
+import { inputBorder } from "./inputBorder";
+import { labelStyles } from "./Label";
+
+const parentStyles = css`
+  width: initial;
+  div {
+    display: flex;
+    :first-child {
+      ${labelStyles}
+    }
+    svg {
+      height: ${space[5]}px;
+      top: 3px;
+      position: relative;
+      right: 30px;
+    }
+  }
+`;
+
+const selectStyles = css`
+  ${inputBorder}
+  font-size: 14px;
+  height: ${space[6]}px;
+  width: initial;
+  padding-right: 50px !important;
+  :hover {
+    cursor: pointer;
+  }
+`;
+
+type CustomDropdownProps = {
+  options: Array<OptionValue<unknown>>;
+  selected: string;
+  label: string;
+  changeHandler: (event: React.ChangeEvent<HTMLSelectElement>) => void;
```

Comment:
I'd argue this should be called `onChange`, as with other elements, as per convention.

Reply:
Good point - will do!